tests/config/test_config.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+"""Document test_config here.
+
+Copyright (C) 2020, Auto Trader UK
+Created 22. Dec 2020 19:10
+
+"""
+import pytest
+
+from olivertwist.config.parser import (
+    ConfigParser,
+    Config,
+    RuleConfig,
+    InvalidConfigException,
+)
+
+
+def test_parsing_valid_config():
+    file_path = "tests/config/valid_config.yml"
+
+    config = ConfigParser.parse(file_path)
+
+    assert config == Config(
+        universal=[
+            RuleConfig(id="no-rejoin-models", enabled=False),
+            RuleConfig(id="no-disabled-models", enabled=True),
+        ]
+    )
+
+
+def test_getting_disabled_rule_ids_from_config():
+    file_path = "tests/config/valid_config.yml"
+
+    config = ConfigParser.parse(file_path)
+
+    assert config.get_disabled_rule_ids() == ["no-rejoin-models"]
+
+
+def test_parsing_invalid_config():
```

Comment:
What do you mean?

Reply:
This actually disappeared with the changes I pushed. But what I meant was:

Instead of
```
def test_parsing_invalid_config():
    with pytest.raises(InvalidConfigException):
        file_path = "tests/config/invalid_config.yml"

        ConfigParser.parse(file_path)
```

it should be

```
def test_parsing_invalid_config():
    file_path = "tests/config/invalid_config.yml"
    
    with pytest.raises(InvalidConfigException):
        ConfigParser.parse(file_path)
```

so it is clear which line is expected to raise the exception.

Reply:
I know it's pedantic in this case as an assignment won't raise an `InvalidConfigException`, but it's a good habit to generally keep indentation to a minimum

Reply:
ah ok